### DISCUSSION TITLE
Captcha Woes

### DISCUSSION THREAD

**Discussion by Ruud68** [Kunena/Kunena-Forum#9589] [Ideas]
Hi with the recent issues with Googles old recaptcha (sites breaking) and J5 removal of all captchaś (except on upgrade from J4), I think it is time to rethink / refactor the current captcha implementation.
This is now implemented based on the assumption that recaptcha is available, but this is nowadays (with J5) not the case anymore.
Kunena should be refactored to use the Global Site configured captcha without adding it's own logic etc.

I am willing to take this task upon me. If this is an option that is. What branch should I develop this against?

**Comment by xillibit** [comment]
Hi,

No problem i let you this change. In this case make your changes on a branch against the branch K6.3

**Comment by Ruud68** [comment]
closed as implemented
